packages/common/src/hooks/useSortBy/useSortBy.stories.tsx

```diff
@@ -19,12 +20,12 @@ const Template: Story = () => {
     <div>
       <div>
         <span> Two buttons to sort by two different keys, ID or Quantity.</span>
-        <button onClick={() => onChangeSortBy({ key: 'id' })}>
+        <BaseButton onClick={() => onChangeSortBy({ key: 'id' })}>
           Sort by ID!
-        </button>
-        <button onClick={() => onChangeSortBy({ key: 'quantity' })}>
+        </BaseButton>
+        <BaseButton onClick={() => onChangeSortBy({ key: 'quantity' })}>
           Sort by Quantity!
-        </button>
+        </BaseButton>
```

Comment:
Using the base button in various stories because I wanted to make the diff really big, apparently.